lib/scope.js

```diff
@@ -1463,4 +1463,13 @@ module.exports = {
     let report = await scope.getPrintableScenario( scope, { scenario });
     _internal_tests.reports( stored_report_key, report );
   },  // Ends scope.testReport()
+
+  testReport2: async function ( scope, { expected_report_text }) {
+    /* Test whether the report is as is expected. A meta test for testing
+    * the behavior of this testing framework. */
```

Comment:
it would be helpful for future development if this explanation included how `testReport` is different from `testReport2`

Reply:
If we use this methodology, `testReport` will be replaced with `testReport2`.